build/buildPackage.sh

```diff
@@ -127,6 +126,15 @@ manifest="
 
 "
 
+if [ "$#" -lt "1" ]; then
```

Comment:
Actually, there's a bunch of other stuff that wouldn't need to be packaged if we weren't supporting renderMan - the lib and python module for IECoreRI for instance. Are you actually trying to make a genuine "RenderMan free" package here, or just stop a build error if the renderMan folder doesn't exist?